x-pack/test/alerting_api_integration/common/fixtures/plugins/actions_simulators/server/plugin.ts

```diff
@@ -126,19 +133,24 @@ export class FixturePlugin implements Plugin<void, void, FixtureSetupDeps, Fixtu
       },
     });
 
-    const router: IRouter = core.http.createRouter();
+    this.router = core.http.createRouter();
 
-    initXmatters(router, getExternalServiceSimulatorPath(ExternalServiceSimulator.XMATTERS));
-    initPagerduty(router, getExternalServiceSimulatorPath(ExternalServiceSimulator.PAGERDUTY));
-    initJira(router, getExternalServiceSimulatorPath(ExternalServiceSimulator.JIRA));
-    initResilient(router, getExternalServiceSimulatorPath(ExternalServiceSimulator.RESILIENT));
-    initMSExchange(router, getExternalServiceSimulatorPath(ExternalServiceSimulator.MS_EXCHANGE));
+    initXmatters(this.router, getExternalServiceSimulatorPath(ExternalServiceSimulator.XMATTERS));
+    initPagerduty(this.router, getExternalServiceSimulatorPath(ExternalServiceSimulator.PAGERDUTY));
+    initJira(this.router, getExternalServiceSimulatorPath(ExternalServiceSimulator.JIRA));
+    initResilient(this.router, getExternalServiceSimulatorPath(ExternalServiceSimulator.RESILIENT));
+    initMSExchange(
+      this.router,
+      getExternalServiceSimulatorPath(ExternalServiceSimulator.MS_EXCHANGE)
+    );
     initServiceNowOAuth(
-      router,
+      this.router,
       getExternalServiceSimulatorPath(ExternalServiceSimulator.SERVICENOW)
     );
   }
 
-  public start() {}
+  public start(core: CoreStart, { actions }: FixtureStartDeps) {
+    initUnsecuredAction(this.router!, actions);
```

Comment:
(I know it's a test plugin, but still) Routes shouldn't be registered during the `start` phase of the plugin.

If you need to access the `actions` start contract from the newly added `/api/sample_unsecured_action` endpoint, you can do so using the `CoreSetup.getStartServices()` API.  


```ts
// inside the endpoint
const [_, { actions }] = await coreSetup.getStartServices()
```

Reply:
Resolved in https://github.com/elastic/kibana/pull/143282/commits/943ac4929a65910189ef5625730f3deeeabd367c in Response Ops PR